test/integration/services.js

```diff
@@ -23,12 +23,12 @@ describe('app', () => {
           expect($('.services').text()).to.equal('Services at this surgery');
           expect($('.services').next('h3').text()).to.equal('Services patients can book themselves');
           expect($('.self-referrals li').length).to.equal(6);
-          expect($($('.self-referrals li')[0]).text()).to.equal('Asthma clinic');
```

Comment:
However the profiles-db-elasticsearch image is no longer used in the deployed application, so there is limited value in testing it.

The service mapping is a feature of the gp-data-etl, which is used by the updater, which updates the elasticsearch cluster, which is rendered on the front end by profiles. 

The best way to verify this would be by acceptance tests against the full stack, rather than in integration tests against the GUI component.